Sources/Testing/Events/Recorder/Event.Symbol.swift

```diff
@@ -22,8 +22,8 @@ extension Event {
     /// The symbol to use when a test passes.
     ///
     /// - Parameters:
-    ///   - knownIssueCount: The number of known issues encountered by the end
-    ///     of the test.
+    ///   - knownIssueCount: The number of known issues recorded for the test.
+    ///     The default value is `0`.
```

Comment:
The only reason there's an associated value here is because, at the time I was building this out, it felt weird to have `case pass` and `case passWithKnownIssues` as separate cases. Doesn't really translate to `case warning`.